docs/source/fx.rst

```diff
@@ -10,7 +10,167 @@ Overview
 Writing Transformations
 -----------------------
 
-TODO
+What is a FX transform? Essentially, it's a function that looks like this.
+
+::
+
+    def transform(m: nn.Module) -> nn.Module:
+        fx_model: GraphModule = fx.symbolice_trace(m)
+        new_model = ...
+        return new_model
+
+Your transform will take in an :class:`torch.nn.Module`, convert it into a
+:class:`GraphModule` with :meth:``symbolic_trace``, and return a new
+``nn.Module``. You should think of the ``nn.Module`` that your FX transform
+returns as identical to a regular ``nn.Module``-- you can pass it to another
+FX transform, you can pass it to TorchScript, or you can
+run it. Ensuring that the inputs and outputs of your FX transform are a
+``nn.Module`` will ensure composability.
+
+Given that you’ve passed in a ``nn.Module`` that has been traced into a
```

Comment:
*an ``nn.Module``

Reply:
I think I actually changed this from correct to incorrect at some point.